src/Pages/ListingPage/ListingPageComponents/CategoryBar/CategoryBar.jsx

```diff
@@ -0,0 +1,40 @@
+import "./CategoryBar.css";
+
+export const CategoryBar = ({ category, setCategory }) => {
+  return (
+    <div className="category-bar-container">
+      <ul className="categories">
+        <li
```

Comment:
suggestion - you can `map` over `categories` array here.

Reply:
sure will think on that